packages/react/src/helpers/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const AMPLIFY_SYMBOL = (typeof Symbol !== 'undefined' &&
```

Comment:
Might be helpful to add a comment why we have this/what this is for. I kinda get it, but I also kinda don't.